datahub-web-react/src/app/entity/shared/containers/profile/sidebar/Domain/SetDomainModal.tsx

```diff
@@ -30,15 +30,14 @@ const StyleTag = styled(Tag)`
     align-items: center;
 `;
 
-export const SetDomainModal = ({ onCloseModal, refetch }: Props) => {
+export const SetDomainModal = ({ urns, onCloseModal, refetch }: Props) => {
     const entityRegistry = useEntityRegistry();
-    const urn = useMutationUrn();
```

Comment:
what happens if the user selects a sibling entity from the multi-select option? Will we know to use the correct urn (ie. the primary sibling)?

Reply:
That's a great question. Right now we merge down multiple results on the same page to a single entity, with the primary sibling showing in the search results. So it should go back to the primary

Reply:
This type of hook should not be in a general purpose modal regardless, as it assumes that its being invoked from an entity profile

Reply:
makes sense! yeah, understood why you took it out of here and wanted to pass an urn in, just wanted to make sure the search modal had you select the primary entity